etl-federation/existing-prometheus/agent-federated.yaml

```diff
@@ -0,0 +1,37 @@
+global:
+  prometheus:
+    enabled: false
+    fqdn: http://prometheus-operated.monitoring:9090
+  grafana:
```

Comment:
Should we also disable Grafana proxy? From my bad memory, if the Grafana proxy is not disabled, it could cause some issues.
```
  grafana:
    enabled: false
    proxy: false
```